relay-general/src/protocol/event.rs

```diff
@@ -227,7 +227,7 @@ pub struct Event {
     pub culprit: Annotated<String>,
 
     /// Transaction name of the event.
-    #[metastructure(max_chars = "culprit")]
+    #[metastructure(max_chars = "culprit", trim_whitespace = "true")]
```

Comment:
Is this enough to trim whitespace from incoming events? The tests assume so.

Reply:
It runs `trim` on the value, removing leading and trailing whitespace -- is that what you want?

Reply:
Yes. Just asking because it happens outside of `process_value` that we call in tests, so we don't test e.g. `transaction: "    "` as input, assuming it will never happen (it will be trimmed beforehand).